README-EN.md

```diff
@@ -54,9 +59,50 @@ dependencies {
 }
 ```
 
-## 2.2 Sync Project
+### 2.1.2 Sync Project
 Click "Sync Project with Gradle Files" button on the Android Studio's menu bar to download dependence files.
 
+
+## 2.2 Eclipse 
+### 2.2.1 Imports SDK jar
+Puts [zplayads.jar](./eclipseJar) into eclipse project's libs，android configure the build path. The main tips as fellows:
+1. Right click the project on eclipse, select Build Path and Configure Build Path...;
+2. Chooses Libraries table, click Add JARs... button;
+3. Select the jar file.
+
+NOTE: The jar contains class and assets fils, it works on eclipse. Don't try put the jar into Android Studio project or Unity project.
```

Comment:
NOTE: zplayads.jar only can be used in Eclipse, if your IDE is Android Studio please refer to 2.1 part.